fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/InsertIntoTableCommand.java

```diff
@@ -140,7 +157,15 @@ public void run(ConnectContext ctx, StmtExecutor executor) throws Exception {
         }
 
         OlapTableSink sink = ((OlapTableSink) planner.getFragments().get(0).getSink());
-
+        if (ctx.getSessionVariable().enableInsertGroupCommit) {
+            // group commit
+            if (!analyzeGroupCommit(sink, physicalOlapTableSink)) {
+                throw new AnalysisException("Nereids does not support group_commit "
+                        + "for this SQL statement, query id: " + ctx.queryId());
+            }
+            handleGroupCommit(ctx, sink, physicalOlapTableSink);
```

Comment:
u could get fragment here, so i think u could reuse legacy planner group commit process code to avoid write same code twice.